docs/research/user-experience/xmr-btc-user-interface-validation.md

```diff
@@ -0,0 +1,49 @@
+# XMR<>BTC Atomic Swap - User Interface Prototype Validation
+
+This document sums up the questions that we would like to validated with 
+
+## Questions
+
+With the UI prototypes we are trying to answer the following questions.
+The questions are split between maker and taker, because the objectives are somewhat different.
+
+Is a `taker` happy to...
+
+1. run a bitcoind full-node
+1. run a monerod full-node
+1. use a webpage for discovering makers
+1. use a privacy chat tool for discovering makers
+1. manually add makers to retrieve orders
```

Comment:
I would not rule it out - this would be the most simple discovery. 
What I was planning to validate is, if users care a lot about having automated discovery. Some might actually prefer using their current privacy preserving tools and then just copy paste. 
Not sure this is relevant though, given what you commented here: https://github.com/comit-network/xmr-btc-swap/pull/43#discussion_r527256451

Reply:
removed it.